mapfishapp/src/main/webapp/app/addons/notes/js/main.js

```diff
@@ -107,12 +107,47 @@ GEOR.Addons.Notes = Ext.extend(GEOR.Addons.Base, {
                 name: "comment",
                 allowBlank: false
             }, {
+                xtype: "checkbox",
+                labelStyle: "width:160px",
+                fieldLabel: this.tr("notes_follow_up"),
+                name: "followup_checkbox",
+                inputValue: true,
+                listeners: {
+                    "change": {
+                        fn: function(checkbox, newValue) {
+                            var followUpHidden = checkbox.findParentByType("form").findBy(function(c) {
+                                return ((c.getXType() === "hidden") &&
+                                (c.name === "followup"));
+                            })[0];
```

Comment:
It's probably better to create a new hidden field instance using the constructor before calling `new Ext.form.FormPanel`. This way you have a ref to it, that you can use here.